traveler-swift-core/TravelerKit/Models/BookingItemSearchParameters.swift

```diff
@@ -95,9 +97,12 @@ struct BookingItemSearchParameters: Decodable {
             self.location = nil
         }
         
-        let iDcategories = try container.decode([String]?.self, forKey: .categories) ?? []
-
-        self.categories = iDcategories.map{ BookingItemCategory(id: $0) }
+        if let categories = try? container.decode([BookingItemCategory].self, forKey: .categories) {
```

Comment:
backend will never send a BookingItemCategory in search parameters. The search parameters are an actual copy of the parameters expected in the booking search endpoint. Therefore you will never get anything else but an id.

Reply:
As discussed, the backend will make the change to always send [String].
I've changed it in the code to always expect [String].